### ISSUE TITLE
After hard drive runs full, CheckforstatusesScheduler job did not recover automatically

### ISSUE THREAD

**Issue by rriemann** [renatolond/mastodon-twitter-poster#810]
Dear all,

the crossposter stopped crossposting (at least from Twitter to Mastodon) and did not recover automatically until the problem was manually identified and the button "retry all" was activated on sidekiq:

![image](https://user-images.githubusercontent.com/111932/185803138-f0cf2f36-22c1-4d91-8c87-2d0c3dbcd5e7.png)

This is most likely a problem caused by a hard drive running out of space, so that some services (DB?) failed. The issue was solved eventually, but the crossposter did not retry again cross posting.

Could the retry rule in sidekiq could be adjusted to just keep on trying?

If the problem is only discovered a few days later, no content is lost for import, but its timestamps will the a few days off.

Best,
Robert

**Comment by renatolond** [comment]
@rriemann I don't believe this job is treated any different than the others, it should follow the regular rules for Sidekiq jobs: retry about 21 times in the period of two weeks before finally giving up. I will test it locally soon.

**Comment by rriemann** [comment]
Maybe after few days the break between attempts grows quickly?

Maybe we can also prevent sidekiq to put this job to the dead queue? See answer here: https://stackoverflow.com/a/23090499/1407622
